jobs/scripts/glusterfs/regression.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+MY_ENV=$(env | sort)
+BURL=${BUILD_URL}consoleFull
+
+# Display all environment variables in the debugging log
+echo "Start time $(date)"
+echo
+echo "Display all environment variables"
+echo "*********************************"
+echo
+echo "$MY_ENV"
+echo
+
+# Remove any gluster daemon leftovers from aborted runs
+sudo -E bash /opt/qa/cleanup.sh
```

Comment:
We don't have a /opt/qa setup on centos ci duffy nodes (which is dynamic allocation). You need to pull in the repo https://github.com/gluster/glusterfs-patch-acceptance-tests (similar to glusterfs) here in the node.

Reply:
Done